apps/client/src/app/pages/accounts/accounts-page.component.ts

```diff
@@ -267,4 +268,21 @@ export class AccountsPageComponent implements OnDestroy, OnInit {
         this.router.navigate(['.'], { relativeTo: this.route });
       });
   }
+
+  private transferCashBalanceDialog(): void {
```

Comment:
Rename to `openTransferCashBalanceDialog()`

Reply:
done